src/server/models/classrooms.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+const mongoose = require("mongoose");
+
+const classroomSchema = new mongoose.Schema({
+    schoolId: { type: "ObjectId", required: false, ref: "School" },
+    courseType: { type: String },
+    courseNumber: { type: String },
+    sectionNumber: { type: String },
+    courseTitle: { type: String, required: true },
+    teachers: [{ type: "ObjectId", required: true, index: true, unique: true, ref: "User" }],
```

Comment:
Remove unique here. it isn't doing what you think its doing. it behaves weirdly on arrays

Reply:
same for TA and student